extensions/scarb-doc/src/lib.rs

```diff
@@ -48,16 +73,22 @@ pub fn generate_packages_information(
             },
         });
     }
-    packages_information
+    Ok(packages_information)
 }
 
 fn generate_language_elements_tree_for_package(
     package_name: String,
     project_config: ProjectConfig,
+    features_cfg: CfgSet,
 ) -> Crate {
-    let db = ScarbDocDatabase::new(Some(project_config));
+    let db: ScarbDocDatabase =
```

Comment:
No need to explicitly type it

Reply:
Vsc extension is putting it as i accidently click the infered type hint xd 
<img width="452" alt="image" src="https://github.com/user-attachments/assets/54dc40e2-0e72-4592-9af3-dc185632b23e">